bundles/org.openhab.binding.speedtest/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="speedtest"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="speedtest">
+		<label>Speedtest Binding</label>
+		<description>Binding for Ookla Speedtest (https://www.speedtest.net/)</description>
+		<channels>
+			<channel id="server" typeId="server"/>
+
+			<channel id="pingJitter" typeId="pingJitter"/>
+			<channel id="pingLatency" typeId="pingLatency"/>
+
+			<channel id="downloadBandwidth" typeId="downloadBandwidth"/>
+			<channel id="downloadBytes" typeId="downloadBytes"/>
+			<channel id="downloadElapsed" typeId="downloadElapsed"/>
+
+			<channel id="uploadBandwidth" typeId="uploadBandwidth"/>
+			<channel id="uploadBytes" typeId="uploadBytes"/>
+			<channel id="uploadElapsed" typeId="uploadElapsed"/>
+
+			<channel id="isp" typeId="isp"/>
+
+			<channel id="interfaceInternalIp" typeId="interfaceInternalIp"/>
+			<channel id="interfaceExternalIp" typeId="interfaceExternalIp"/>
+
+			<channel id="resultUrl" typeId="resultUrl"/>
+
+			<channel id="triggerTest" typeId="triggerTest"/>
+		</channels>
+		<properties>
+			<property name="Server List 1"></property>
+			<property name="Server List 2"></property>
+			<property name="Server List 3"></property>
+			<property name="Server List 4"></property>
+			<property name="Server List 5"></property>
+			<property name="Server List 6"></property>
+			<property name="Server List 7"></property>
+			<property name="Server List 8"></property>
+			<property name="Server List 9"></property>
+			<property name="Server List 10"></property>
+		</properties>
+		<config-description>
+			<parameter-group name="config-info">
+				<label>Speedtest Configuration</label>
+			</parameter-group>
+			<parameter name="refreshInterval" type="integer" required="true" groupName="config-info">
+				<label>Refresh Rate</label>
```

Comment:
```suggestion
				<label>Refresh Interval</label>
```
It is less confusing if the name and label are the same. Some people use textual and others use the UI so in forum posts you can call it X and its accurate for both methods.

Reply:
done